From 7d786fc7ebe6110fa33099d7c38839eb02b71d85 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Tue, 1 Jun 2021 13:02:37 +0200
Subject: [PATCH] imx8mm-var-dart-plt: Switch dtb name for kernel 5.4.3

We have switched to kernel 5.4.3 which uses a different dtb name so
let's use the new name here.

Upstream-status: Inappropriate[configuration]
Signed-off-by: Florin Sarbu <florin@balena.io>
---
 include/configs/imx8mm_var_dart.h | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/include/configs/imx8mm_var_dart.h b/include/configs/imx8mm_var_dart.h
index 12c67ee303..138cfadba0 100644
--- a/include/configs/imx8mm_var_dart.h
+++ b/include/configs/imx8mm_var_dart.h
@@ -116,11 +116,7 @@
 					"setenv fdt_file imx8mm-var-som-symphony.dtb; " \
 				"fi; " \
 			"else " \
-				"if test $carrier_rev = legacy; then " \
-					"setenv fdt_file imx8mm-var-dart-dt8mcustomboard-legacy.dtb; " \
-				"else " \
-					"setenv fdt_file imx8mm-var-dart-dt8mcustomboard.dtb; " \
-				"fi; " \
+				"setenv fdt_file imx8mm-var-dart.dtb; " \
 			"fi; " \
 		"fi; \0" \
 	"loadfdt=run findfdt; " \
-- 
2.17.1

